packages/scripts/README.md

```diff
@@ -0,0 +1,39 @@
+# molgenis-scripts
```

Comment:
please add short description of functionality, maybe point to docs for further info